stdlib/2and3/_heapq.pyi

```diff
@@ -11,5 +11,7 @@ def heappush(heap: List[_T], item: _T) -> None: ...
 def heappushpop(heap: List[_T], item: _T) -> _T: ...
 def heapreplace(heap: List[_T], item: _T) -> _T:
     raise IndexError()  # if list is empty
-def nlargest(a: int, b: List[_T]) -> List[_T]: ...
-def nsmallest(a: int, b: List[_T]) -> List[_T]: ...
+def nlargest(n: int, iterable: Iterable[_T],
+             key: Optional[Callable[[_T], Any]] = ...) -> List[_T]: ...
+def nsmallest(n: int, iterable: Iterable[_T],
+              key: Optional[Callable[[_T], Any]] = ...) -> List[_T]: ...
```

Comment:
The Python 3 version of `_heapq` does not seem to have either of these, so they need to be behind a version check.